contracts/strategy/StrategyBaseV2.sol

```diff
@@ -146,10 +154,15 @@ abstract contract StrategyBaseV2 is IStrategyV2, ControllableV3 {
   // *************************************************************
 
   /// @dev Stakes everything the strategy holds into the reward pool.
-  function investAll() external override {
+  /// @param amount_ Amount transferred to the strategy balance just before calling this function
+  function investAll(uint amount_) external override {
```

Comment:
rename this function for more properly name

Reply:
I can imagine only ugly complex names like
addAmountAndInvestAll
because actually this function does two things:
1) register new amount in baseAmounts
2) investAll
I don't like names with and, probably it worth to continue to use investAll and make more descriptive name of the parameter, i.e. amount_ => amountAddToStratagyBalance_